client/a8c-for-agencies/data/sites.ts

```diff
@@ -0,0 +1,301 @@
+export const mockedSites = [
```

Comment:
```suggestion
export const mockedSites: Site[] = [
```

If you change this, this will also highlight which values need changing from the const, and changing those should also prevent the currently failing unit tests:
```
Type '{ url: string; url_with_scheme: string; blog_id: number; blogname: string; audit_updated_at: string; sticker: string[]; public: boolean; is_connection_healthy: boolean; has_backup: boolean; ... 24 more ...; monitor_last_status_change?: undefined; } | { ...; }' is not assignable to type 'Site'.
Type '{ url: string; url_with_scheme: string; blog_id: number; blogname: string; audit_updated_at: string; sticker: string[]; public: boolean; is_connection_healthy: boolean; has_backup: boolean; has_scan: boolean; ... 23 more ...; monitor_last_status_change?: undefined; }' is not assignable to type 'Site'.
 The types of 'monitor_settings.monitor_site_status' are incompatible between these types.
Type 'null' is not assignable to type 'boolean'.
```

Reply:
Thank you Karen!

Reply:
I'll add a commit to fix this.